apps/backend/.env.development

```
@@ -78,7 +78,7 @@ STACK_STRIPE_SECRET_KEY=sk_test_mockstripekey
 STACK_STRIPE_WEBHOOK_SECRET=mock_stripe_webhook_secret
 STACK_OPENROUTER_API_KEY=FORWARD_TO_PRODUCTION
 STACK_FEEDBACK_MODE=FORWARD_TO_PRODUCTION
-# STACK_DOCS_INTERNAL_BASE_URL=http://localhost:8104
+STACK_MINTLIFY_MCP_URL=https://stackauth-e0affa27.mintlify.app/mcp
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STACK_MINTLIFY_MCP_URL key should go before the STACK_OPENAI_API_KEY key

(UnorderedKey)